coi/src/utils.rs

```diff
@@ -111,6 +114,17 @@ pub(crate) mod serde_duration_as_days {
     }
 }
 
+/// # Panics
+///
+/// This method is primarily used for testing.
+/// It simply converts an Array into a normalized Array.
+///
+/// Will panic if the array cannot be normalized.
+pub fn normalize_array<const N: usize>(array: [f32; N]) -> [f32; N] {
+    let embedding: Embedding<Ix1> = Array::from_vec(array.into()).into();
+    embedding.normalize().unwrap().to_vec().try_into().unwrap()
```

Comment:
:art: we could still simplify this by (importing `use xayn_ai_bert::Embedding1;` instead)
```suggestion
    Embedding1::from(array)
        .normalize()
        .unwrap()
        .to_vec()
        .try_into()
        .unwrap()
```